packages/components/psammead-caption/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-caption",
-  "version": "4.0.5",
+  "version": "4.1.0",
```

Comment:
Could we bump the `package-lock.json` as well when we bump `package.json`?

Reply:
Thanks Fahad! I've committed those changes 🙂